api-scripts/inventory-storage/inventoryStorage_db_pool_utilization/README.md

```diff
@@ -0,0 +1,31 @@
+## Modules required to be enabled as a pre-requisite to run JMeter script:
+-mod-inventory-storage
+-mod-authtoken
+-mod-permissions
+-okapi
+
+## Usage
+- This inventoryStorage_db_pool_utilization.jmx script that invokes the GET /instance-storage/instances/${instanceId} API. It uses a list of instances from instances.csv 
+- Modify credentials.csv to have the correct values before running the script.
+
+## Workflow steps:
+
+##### Workflow for inventoryStorage_db_pool_utilization.jmx
+1. Open script and edit the quantity of thread groups in accordance with amount of tenants and hosts (servers).
```

Comment:
"edit the quantity of thread groups". Could you be more explicit by stating something like "to add more thread groups copy and paste the existing thread group and give them unique names".